pyriemann/featureselection.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+from .utils.distance import distance_riemann
+from .utils.mean import mean_riemann
+
+
+def class_distinctiveness(C_class1, C_class2):
```

Comment:
```suggestion
def class_distinctiveness(C_class1, C_class2):
```

I would suggest to have an API that works for more than 2 classes. So something like

`class_distinctiveness(X, y)`

where y are class labels and X is an array of covariances